gnomad/resources/grch38/gnomad.py

```diff
@@ -472,9 +545,11 @@ def gnomad_gks(
     # Read coverage statistics if requested
     if high_level_version == "v3":
         coverage_version = "3.0.1"
+    elif high_level_version == "v4":
+        coverage_version = "3.0.1"
```

Comment:
@klaricch is there just exomes coverage available in v4? That's all I see in the gnomad downloads page and in the `coverage` function. If coverage info is requested for a v4 genomes table, what should the behavior be?

Reply:
ah! gnomAD v4 genomes are the same as gnomAD v3 genomes , so the coverage ought to be the same for v4 genomes as for v3 genomes.  This might require some extra logic with the user selecting a data type - I forget if that is already requested